### ISSUE TITLE
[BUG] Missing Config Folder

### ISSUE THREAD

**Issue by echo524** [aliparlakci/bulk-downloader-for-reddit#312]
- [X] I am reporting a bug.
- [x] I am running the latest version of BDfR
- [X] I have read the [Opening an issue](README.md#configuration)
  
## Description
On Windows 10, after install and even successfully running the tool multiple times, there isn't a config folder created. There are no log files that I can find or the config file. Using verbose (see below), I see it loads a config file, but looking in that location neither the BDFR or BDFR\bdfr\ folder is there. Picture of AppData\Local\ attached below.

## Command
`python3 -m bdfr download -v --authenticate --user me --upvoted E:\NewTest`

## Environment (please complete the following information):
 - OS: Windows 10 20H2
 - Python version: 3.9.4

## Logs
```
C:\Users\eprice>python3 -m bdfr download -v --authenticate --user me --upvoted E:\NewTest
[2021-04-27 23:42:13,789 - bdfr.downloader - DEBUG] - Loading configuration from C:\Users\eprice\AppData\Local\BDFR\bdfr\default_config.cfg
[2021-04-27 23:42:13,800 - bdfr.downloader - DEBUG] - Setting maximum download wait time to 120 seconds
[2021-04-27 23:42:13,801 - bdfr.downloader - DEBUG] - Using authenticated Reddit instance
```

![2021-04-27_23-48-27](https://user-images.githubusercontent.com/75281881/116344558-6ecfdc00-a7b4-11eb-90a0-de16283ac49f.png)

**Comment by Serene-Arc** [comment]
@echo524 I can't recreate this, the BDFR makes the folder as expected whenever I run the program. Can you confirm that this is still an issue?

**Comment by echo524** [comment]
Yeah, I still have this. Even uninstalled release and installed the development branch and the folder still doesn't create.

Could it have anything to do with the destination being another drive (this is a flash drive)?

Verbose still shows it pulls the config file from the same place (C:\Users\eprice\AppData\Local\BDFR\bdfr\default_config.cfg), but the folder is not there. Could it be pulling from the default_config.cfg in the source folder?

**Comment by echo524** [comment]
Hey guys, I did find them finally. Did a search for the log_output.txt and found both the log files and config here:
`C:\Users\eprice\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\Local\BDFR\bdfr`

It looks like python has its own LocalCache that thinks it's the Local folder or something like that.
I think I know why actually, my Python 3.9 is download from the Windows Store, that might cause the reference to go somewhere else.

I'll go ahead and close this then, but let me know if you want me to test something with this setup instead.
Thanks again for the help!

**Comment by aliparlakci** [comment]
You are awesome! Thanks a lot. This was a pretty edge case. I will keep the issue open until we figure out what is really happening.
